tests/test_dist_to_ports.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+"""
+Distance to Ports tests
+"""
+from click.testing import CliRunner
+import json
+import numpy as np
+import os
+import pytest
+import shutil
+import tempfile
+import traceback
+
+from rex.utilities.loggers import LOGGERS
+from reV.handlers.exclusions import ExclusionLayers
+from reVX import TESTDATADIR
+from reVX.offshore.dist_to_ports import DistanceToPorts
+from reVX.offshore.dist_to_ports_cli import main
+
+EXCL_H5 = os.path.join(TESTDATADIR, 'offshore', 'dist_to_coast.h5')
+PORTS_FPATH = os.path.join(TESTDATADIR, 'offshore', 'ports',
+                           'ports_operations.shp')
+
+
+def get_dist_to_ports(excl_h5, ports_layer='ports_operations'):
+    """
+    Extract "truth" distance to ports layer from exclusion .h5 file
+    """
+    with ExclusionLayers(excl_h5) as f:
+        dist_to_ports = f[ports_layer]
+
+    return dist_to_ports
+
+
+@pytest.fixture(scope="module")
+def runner():
+    """
+    cli runner
+    """
+    return CliRunner()
+
+
+@pytest.mark.parametrize('max_workers', [None, 1])
+def test_dist_to_ports(max_workers):
+    """
+    Compute distance to ports
+    """
+    baseline = get_dist_to_ports(EXCL_H5)
+    dist = DistanceToPorts(PORTS_FPATH, EXCL_H5)
+    test = dist.least_cost_distance(max_workers=max_workers)
+
+    msg = 'distance to ports does not match baseline distances'
+    assert np.allclose(baseline, test), msg
+
+
+@pytest.mark.parametrize('ports_layer', [None, 'test'])
+def test_cli(runner, ports_layer):
```

Comment:
I thought you were going to keep the heatmap plot output as an option in the test for this feature? This is definitely a feature thats easier to visualize.

Reply:
brain fart, I'll add in that function.